File: C Programming/Competitive Programming/Coding Challenges 8/program3.c
/*3. Write a program which accept distance in kilometre and convert it into meter. (1 kilometre = 1000 Meter)
Input: 5
Output: 5000

Input: 12
Output: 12000
*/
#include<stdio.h>

int KMtoMeter(int iNo)
{
    int iMult =0;
    iMult=iNo*1000;
    return iMult;
}
int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter distance: ");
    scanf("%d",&iValue);

    iRet = KMtoMeter(iValue);

    printf("Distance is meter is : %d",iRet);

    return 0;

}

File: Java Programming/Competitive Programming/Coding Challenges 16/program3.java
/*
3. Write a program which accept range from user and return addition of all numbers in between that range. (Range should contains positive numbers only)
Input : 23 30
Output : 212
Input : 10 18
Output : 126
Input : -10 2
Output : Invalid range
Input : 90 18
Output : Invalid range
*/

import java.util.Scanner;

class program3
{
    static int RangeSum(int iStart, int iEnd)
    {
        if ((iStart < 0) || (iEnd < 0) || (iStart > iEnd))
        {
            return -1;
        }

        if (iStart == iEnd)
        {
            return iStart;
        }

        return iStart + RangeSum(iStart + 1, iEnd);
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue1 = 0;
        int iValue2 = 0;
        int iRet = 0;

        System.out.print("Enter starting point : ");
        iValue1 = sobj.nextInt();

        System.out.print("Enter ending point : ");
        iValue2 = sobj.nextInt();

        iRet = RangeSum(iValue1, iValue2);

        if (iRet == -1)
        {
            System.out.print("Invalid Range");
        }
        else
        {
            System.out.print("Addition is : " + iRet);
        }

        sobj.close();
    }
}